hbase_sql/src/main/java/com/splicemachine/stream/StreamableRDD.java

```diff
@@ -43,12 +55,12 @@
     private final int clientBatchSize;
     private final String host;
     private final JavaRDD<T> rdd;
-    private final ExecutorCompletionService<Object> completionService;
     private final ExecutorService executor;
     private final int clientBatches;
     private final UUID uuid;
     private final OperationContext<?> context;
     private OlapStatus jobStatus;
+    private BlockingQueue<Future<Object>> futures = new LinkedBlockingDeque<>();
```

Comment:
Is the use of LinkedBlockingDeque instead of LinkedBlockingQueue on purpose?

Reply:
No, initially I tried a Deque but I realized it wasn't needed.